core/arch/arm/plat-mediatek/entry_a64.S

```diff
@@ -0,0 +1,307 @@
+/*
```

Comment:
I guess for the near future we should do something here, this file and ./core/arch/arm/plat-vexpress/entry_a64.S are almost identical.